conbench/job.py

```diff
@@ -93,7 +114,7 @@ class CacheDict(TypedDict):
 
 # Fetching one million items from a sample DB takes ~1 minute on my machine
 # (the `results = Session.scalars(....all())` call takes that long.
-def _fetch_and_cache_most_recent_results(n=0.08 * 10**6) -> None:
+def _fetch_and_cache_most_recent_results(n=0.05 * 10**6) -> None:
```

Comment:
Oh, I want to revert that to keep changes in plots meaningful. Commit coming. That was just for testing.